contracts/axelar-gas-service/tests/test.rs

```diff
@@ -347,18 +333,21 @@ fn fail_refund_not_enough_balance() {
 
     let message_id = message_id(&env);
 
-    assert!(client.try_refund(&message_id, &receiver, &token).is_err());
+    assert!(client
+        .mock_all_auths()
+        .try_refund(&message_id, &receiver, &token)
+        .is_err());
```

Comment:
is this because the error is from transfer method? can we use should_panic to test the error message rather than an arbitrary error